### PR TITLE
Update build system and development instructions

### LINKED ISSUE #767: building cozy on fedora
i am working on fedora and so i had to find the packages that need to be installed there.

https://gitlab.com/eMBee/cozy/-/commit/0ae6e20a929a8e24b0651c4b901eb2438da146e1

it may be useful to add that in the documentation.

### PR DESCRIPTION
**Pull request by rdbende**
Marked as draft as it depends on #799.

Resolves #767

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -21,23 +21,14 @@ sudo apt install \
...
+  libadwaita-1-dev
   pip \
   python-gi-cairo \
   python3-gst-1.0 \
   python3-venv
-
-sudo add-apt-repository ppa:apandada1/libhandy-1
-sudo apt update
-sudo apt install libhandy-1-0 libhandy-1-dev
 ```
```
Comment (rdbende) [inline DEVELOPMENT.md:30]: Note: I didn't check whether the updated packages work on Ubuntu, as I'm on Fedora, but they should.

**Review by rdbende — COMMENTED** [review]
TODO:

- [ ] Add section for building with GNOME Builder
- [x] Add package list for Fedora (#767)

File: DEVELOPMENT.md (hunk 1 above)
Comment (geigi) [reply DEVELOPMENT.md:43]: The packages should be fine. One backslash is gone missing after `libadwaita-1-dev`, could you fix that one?

Comment (rdbende) [reply DEVELOPMENT.md:43]: Good catch, thanks!

**Pull request merged**